### ISSUE TITLE
Feature Request: Hide folders

### ISSUE THREAD

**Issue by shenom-x** [protz/Manually-Sort-Folders#77]
As I can´t get rid of some folders and some are not needed (by myself) - this woukd be a nice little extra feature.

Or does somebody know another addon? Can´t find anything yet.

Thx / Sascha

**Comment by Gluups** [comment]
Never experimented any difficulty to suppress a folder.
What type of folder ?
What is the link with "Manually sort folders" ?

**Comment by mschwerhoff** [comment]
With IMAP (maybe with POP3 as well?) you can simply unsubscribe from certain folders in Thunderbird. Right-click your account, then choose "Subscribe...".

**Comment by jason-hub** [comment]
That is true, Malte, but some folders mysteriously return even after unsubscribing.

**Comment by mschwerhoff** [comment]
Did you try deleting the "cache" files that Thunderbird creates for IMAP folders? Check your Thunderbird profile folder, then probably `ImapMail/<your-account>`. There, you'll see files and directories named after your IMAP folders. Delete them (or, to be on the safe side, temporarily move them), restart Thunderbird and hope that the unwated folders are gone for good.

**Comment by think-info** [comment]
While both sides of the above argument are correct, there is one detail missing.
Yes, we can unsubscribe from IMAP folders and - at least in theory - hide them that way, But we cannot unsubscribe from Local Folders… as in the built-in account. A hide function - at least on that account - would be very useful.

**Comment by lgbaldoni** [comment]
>  But we cannot unsubscribe from Local Folders

Is there a way to delete them?

**Comment by datenheim** [comment]
@lgbaldoni 
At least in TB115.10 there is a "Hide local Folders" option in the ... menu of the tree view
